src/executables/javasp.cpp

```diff
@@ -366,14 +370,16 @@ javasp_status_server (const JAVASP_SERVER_INFO jsp_info)
   if (socket != INVALID_SOCKET)
     {
       char *ptr = NULL;
-      OR_ALIGNED_BUF (OR_INT_SIZE * 2) a_request;
+      OR_ALIGNED_BUF (OR_INT_SIZE * 4) a_request;
       char *request = OR_ALIGNED_BUF_START (a_request);
 
-      ptr = or_pack_int (request, SP_CODE_UTIL_STATUS);
+      ptr = or_pack_int (request, OR_INT_SIZE);
+      ptr = or_pack_int (ptr, SP_CODE_UTIL_STATUS);
+      ptr = or_pack_int (ptr, OR_INT_SIZE);
       ptr = or_pack_int (ptr, SP_CODE_UTIL_TERMINATE_THREAD);
 
-      int nbytes = jsp_writen (socket, request, (int) sizeof (int) * 2);
-      if (nbytes != (int) sizeof (int) * 2)
+      int nbytes = jsp_writen (socket, request, (int) sizeof (int) * 4);
+      if (nbytes != (int) sizeof (int) * 4)
```

Comment:
How about writing OR_INT_SIZE instead of "(int) sizeof (int)"?
Of course, it is the same size, but OR_INT_SIZE  is also used inside or_pack_int().
It would be nice to use the same convention throughout the code.

Reply:
It's a good point! thank you